src/screens/Assistant/ResultItem.tsx

```diff
@@ -44,11 +44,13 @@ import {
 import {ScrollView} from 'react-native-gesture-handler';
 import {Leg, TripPattern} from '@atb/api/types/trips';
 import {Mode} from '@atb/api/types/generated/journey_planner_v3_types';
+import {SearchTime} from './journey-date-picker';
 
 type ResultItemProps = {
   tripPattern: TripPattern;
   onDetailsPressed(): void;
   testID?: string;
+  searchTime?: SearchTime;
```

Comment:
Må denne være optional? Pluss at den kan nok legges over `testID`. Sikkert greit prinsipp å ha test-parametre nederst for et tydeligere skille mellom "virkelige" parametre og parametre til test.

Reply:
Her har vi tenkt litt motsatt ser jeg. Jeg har tenkt "må denne være required?", slik at hvis ResultItem skal brukes et annet sted, trenger man ikke å oppgi SearchTime mot at det ikke får mulighet for gjennomstreking. Ser ikke for meg at den kommer til å bli gjenbrukt med det første, så forandrer det gjerne til required, men skal jeg tenke "må denne være optional?" om parametere neste gang? 

Enig i at den bør ligge over testID 👍

Reply:
Jeg liker bedre at de er required hvis det er mulig. Om de er optional så må man støtte to alternativer på en god måte, både at den blir sendt inn og at den ikke blir sendt inn. Ikke store greier, spesielt ikke her, men mange bekker små osv...

Reply:
Den er god! Ikke en stor sak, nei, men fint å ha det avklart 👍

Reply:
Et eksempel på dette (som blir teoretisk for dette prosjektet) er at om man hadde unit tester på komponent-nivå så måtte man unit-testet begge flyter, mens om den er required klarer den seg med en.